### PR TITLE
[ADD] website_portal: Restrict modif personal info

### PR DESCRIPTION
**Pull request by remytms**
Restrict the modification of personal info in the website portal.

This is based on the website_portal_extend of Easy My Coop. This module is not merged yet. See : https://github.com/houssine78/vertical-cooperative/pull/2

So don't merge before the module website_portal_extend is merged in Easy My Coop.

### CODE REVIEW

**Comment by remytms** [comment]
website_portal_extend has been merged in Easy My Coop.

**Comment by remytms** [comment]
A review is welcome.

File: beesdoo_website_portal/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'BEES coop Website Portal',
+
```
Comment (houssine78) [inline beesdoo_website_portal/__openerp__.py:8]: Choose a more generic module name. This module may be installed for other cooperative supermarkets

File: beesdoo_website_portal/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# Copyright 2017-2018 Rémy Taymans <remytaymans@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp.addons.website_portal_extend.controllers.main import ExtendWebsiteAccountController
+
+
+class BeesdooAccountWebsiteController(ExtendWebsiteAccountController):
+
```
Comment (houssine78) [inline beesdoo_website_portal/controllers/main.py:11]: same remark goes here

**Review by houssine78 — APPROVED** [review]

File: beesdoo_website_portal/__openerp__.py (hunk 1 above)
Comment (remytms) [reply beesdoo_website_portal/__openerp__.py:8]: We can speak about that. But the purpose of this module is to restrict the possibility to modify some information of the user. This module is very "stupid", so it's designed to have one module for each client that need to restrict something in the personal user form.

If needed we can create a generic module, if we see that many clients want the same restriction.

**Pull request merged**